packages/compass-aggregations/src/components/stage-preview/stage-preview.module.less

```diff
@@ -66,26 +66,36 @@
     display: flex;
     align-items: stretch;
     overflow-x: scroll;
-    margin: 10px;
-    padding: 10px;
```

Comment:
This was margin + padding for a reason so that scrollbar doesn't stick weirdly to the bottom of the card and has some spacing there, so maybe worth keeping this

Reply:
Ah thanks for the context. Re-added it. I updated it to 8px each so the total is 16px, which is one of the leafygreen `token` `spacing` values. Do you like the 20px here as well or is the 16px better?